### PR TITLE
Update build instructions in README.md

### PR DESCRIPTION
**Pull request by gioneill**
Clarify some sticking points on the cURL and openSSL config steps.

### CODE REVIEW

**Comment by winniequinn** [comment]
@gioneill I reworked the instructions for cURL and OpenSSL before merging. If you have any objections, please bonk me or make a new PR!

**Comment by gioneill** [comment]
No objections!

**Comment by winniequinn** [comment]
@gioneill If you can, please rename the branch before merging or make sure to adjust the merge title appropriately when merging. It'll be hard to remember what `gioneill-patch-1` was later when looking at commits in `master`.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
```
Comment (daryanypl) [inline README.md]: Do we want to be helpful with a sample command?  "ln -s [your_path_here/]DRM_Connector_Prerelease adobe-rmsdk"

File: README.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
```
Comment (daryanypl) [inline README.md]: I also installed the Command Line Tools, while I was trying to figure out how to build libcurl.  I think I remember you telling me that I needed to install something with XCode that I didn't have, but I don't remember what it was.  After the curl got eliminated, the project just built, and I didn't need to install anything else, so I'm wondering if the Command Line Tools were a requirement.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
-07. Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3).*
+07. Build OpenSSL and cURL as described in the following "Building OpenSSL and cURL" section. Ensure you're in the "Simplified-iOS" directory before continuing to the next step.
 08. `sh adobe-rmsdk-build.sh`
```
Comment (daryanypl) [inline README.md]: is it a slash after "rmsdk"?

File: README.md [hunk 4]
```diff
@@ -1,18 +1,26 @@
...
 09. `cp APIKeys.swift.example Simplified/APIKeys.swift` and edit accordingly.
 10. `open Simplified.xcworkspace`
 11. Build.
 
-\* If you want to release your app in Apple's App Store, you must add `--enable-ipv6` to line 80 of Adobe's "build.sh" script used for building cURL. Without that flag, cURL will not provide proper support for IPv6 and therefore Apple will reject your application.
+## Building OpenSSL and cURL
+
```
Comment (daryanypl) [inline README.md]: I have a note saved: "don’t gunzip, build.sh script wants to do that for you".

File: README.md [hunk 5]
```diff
@@ -1,18 +1,26 @@
...
+Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3). Be sure to note the following:
+
+* You will need to verify and edit the "build.sh" scripts for both OpenSSL and cURL to reflect the correct version numbers and local directory names (lines 11–24).
+* You must add `--enable-ipv6` to line 80 of Adobe's "build.sh" script used for building cURL. This necessary both due to Apple's requirements for IPv6 support and because the library may not build against recent iOS SDKs otherwise.
```
Comment (daryanypl) [inline README.md]: I have notes saved: "values to edit in build.sh before building:
OPENSSL_VERSION=“1.0.2m”
SDK_VERSION
min_version

tell them to make the public/ios directory to copy the include and lib into"

File: README.md [hunk 6]
```diff
@@ -1,18 +1,26 @@
...
+* cURL 7.57.0 is known _not_ to work and later versions are unlikely to work either. 7.50.0 is recommended.
```
Comment (daryanypl) [inline README.md]: remember to edit the build.sh to match the openassl one

File: README.md [hunk 7]
```diff
@@ -1,18 +1,26 @@
...
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
-07. Follow the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf" to build OpenSSL (section 12.1) and cURL (section 12.3).*
+07. Build OpenSSL and cURL as described in the following "Building OpenSSL and cURL" section. Ensure you're in the "Simplified-iOS" directory before continuing to the next step.
 08. `sh adobe-rmsdk-build.sh`
 09. `cp APIKeys.swift.example Simplified/APIKeys.swift` and edit accordingly.
```
Comment (daryanypl) [inline README.md]: I have a note saved: "for step 9, we need to explain where APIKeys.swift.example is to be coming from", so I'm guessing it didn't just appear in the file, I probably had to download it from a private repository?

**Pull request closed without merging**